### DISCUSSION TITLE
Possible bug? Wallos LXC update

### DISCUSSION THREAD

**Discussion by Aloe-recite** [community-scripts/ProxmoxVE#78] [Q&A]
Hi all,
Wallos LXC is giving some troubles when using the "Update" command, it says that it's all well (suspiciously fast maybe?) but if I reboot the container and open the app it is still on the previous version. Manual update (i.e. copying files over) works.
Can someone please check and confirm if it's a bug on my end or something general?
Thanks

**Comment by MickLesk** [comment]
Please use the issues Section to create an BUG
